packages/components/pie-toggle-switch/src/index.ts

```diff
@@ -84,6 +85,7 @@ export class PieToggleSwitch extends RtlMixin(LitElement) implements ToggleSwitc
                     ?isChecked="${isChecked}"
                     ?isDisabled=${isDisabled}>
                     <input
+                        id="toggle-switch"
```

Comment:
nothing weird happens if we have multiple toggle switches on a single page all with the same ID, does it?